src/other/WordCloud.js

```diff
@@ -5,7 +5,7 @@
     } else {
         root.other_WordCloud = factory(root.d3, root.common_SVGWidget, root.other_IWordCloud);
     }
-}(this, function (d3, SVGWidget, IWordCloud) {
+}(this, function (d3, SVGWidget, IWordCloud, d3Cloud) {
```

Comment:
if not defined there you will get an error about d3Cloud not being defined below